<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QFDlgCSVParameters</class>
 <widget class="QDialog" name="QFDlgCSVParameters">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>944</width>
    <height>534</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>539</width>
    <height>229</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CSV Import Properties</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,1,0">
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Please specify the parameters used to import a comma separated values (CSV) file into QuickFit.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QVisibleHandleSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Parameters:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QConfigComboboxWidget" name="widConfig" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>&amp;Column Separator:</string>
           </property>
           <property name="buddy">
            <cstring>edtColumn</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="edtColumn">
           <property name="text">
            <string>,</string>
           </property>
           <property name="maxLength">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="btnTab">
             <property name="text">
              <string>set tab</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>&amp;Decimal Separator:</string>
           </property>
           <property name="buddy">
            <cstring>edtDecimal</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="edtDecimal">
           <property name="text">
            <string>.</string>
           </property>
           <property name="maxLength">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>C&amp;oment Separator:</string>
           </property>
           <property name="buddy">
            <cstring>edtComment</cstring>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="edtComment">
           <property name="text">
            <string>#</string>
           </property>
           <property name="maxLength">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>&amp;Header Start:</string>
           </property>
           <property name="buddy">
            <cstring>edtHeader</cstring>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="edtHeader">
           <property name="text">
            <string>#!</string>
           </property>
           <property name="maxLength">
            <number>100</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>File contents:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>guess CSV parameters</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="txtContents">
         <property name="font">
          <font>
           <pointsize>7</pointsize>
          </font>
         </property>
         <property name="lineWrapMode">
          <enum>QPlainTextEdit::NoWrap</enum>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="backgroundVisible">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Result:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>reparse test-data</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="tableView"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVisibleHandleSplitter</class>
   <extends>QSplitter</extends>
   <header>qvisiblehandlesplitter.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QConfigComboboxWidget</class>
   <extends>QWidget</extends>
   <header>qconfigcomboboxwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QFDlgCSVParameters</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>524</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>QFDlgCSVParameters</receiver>
   <slot>reloadCSV()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>918</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>940</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>QFDlgCSVParameters</receiver>
   <slot>guessParameters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>855</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>941</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
 <slots>
  <slot>reloadCSV()</slot>
  <slot>guessParameters()</slot>
 </slots>
</ui>
